### ISSUE TITLE
Magic (plugin) doesn't work with playervaults (with fix)

### ISSUE THREAD

**Issue by Floffah** [drtshock/PlayerVaults#395]
### PlayerVaultsX Version
*Run `/version playervaults` and give us your output.*
The latest from this repo (pom.xml says 4.1.4)

### Server Version
*Run `/version` and give us your output.*
Latest paper build #172 (not an issue that is the build number) (but the issue was tested on the latest spigot too)

### Logs
*Give use your logs by going to `logs/latest.log`.*
i don't have access

### Details
*Please explain the problem.*
When a user puts a magic wand into a player vault they can't open the pv unless someone removes it from their user data

i had to create a fix for my server.
i changed saveVault in listeners/Listeners to
```java
public void saveVault(Player player, Inventory inventory) {
        if (plugin.getInVault().containsKey(player.getUniqueId().toString())) {

            Inventory inv = Bukkit.createInventory(null, 6 * 9);
            inv.setContents(inventory.getContents().clone());
            ItemStack[] stacks = inv.getContents();
            for (int i = 0; i < stacks.length; ++i) {
                ItemStack item = stacks[i];
                if (item != null) {
                    if (item.getType().equals(Material.WOODEN_HOE)) {
                        stacks[i] = null;
                        player.getInventory().addItem(item);
                        player.sendMessage(ChatColor.translateAlternateColorCodes('&', "&4Due to conflicts in plugins, you can no longer put wands in chests."));
                    }
                }
            }
            inv.setContents(stacks);
            if (inventory.getViewers().size() == 1) {
                VaultViewInfo info = plugin.getInVault().get(player.getUniqueId().toString());
                vaultManager.saveVault(inv, info.getVaultName(), info.getNumber());
                plugin.getOpenInventories().remove(info.toString());
            }

            plugin.getInVault().remove(player.getUniqueId().toString());
        }
    }
```

**Comment by xH3LLRAIZ3Rx** [comment]
I don't seem to get this issue, odd

**Comment by Floffah** [comment]
turns out the wands came from magic not slimefun big oof but it still doesn't work and the fix works for me

**Comment by CmdrKittens** [comment]
This should be resolved in 4.1.6, if you're able to test it.
